src/generator/AutoRest.Ruby/Templates/EnumTemplate.cshtml

```diff
@@ -8,7 +8,7 @@ module @Settings.Namespace
     #
     @WrapComment("# ", "Defines values for " + Model.Name)
     #
-    module @(Model.Name)
+    module @(CodeNamer.Instance.PascalCase(Model.Name))
```

Comment:
Ohhh i though fix should be inside then the template as the same names are being used in requirement templates as well :(